### ISSUE TITLE
Clarification on whether changing the error here is considered as making it more general

### ISSUE THREAD

**Issue by clarud** [nus-cs2103-AY2425S1/forum#686]
One of our tester identified an error in our error message where we specify the error causing field as such
![image](https://github.com/user-attachments/assets/d360cbfa-5b04-400c-87c7-9fb17bf695ba)
It is supposed to show a more general error message such as shown below
![image](https://github.com/user-attachments/assets/0b37c077-327c-4eae-a30a-29595f7f8ebb)

Will this be considered Q6 on widening scope of message or is this change not allowed?

**Comment by damithc** [comment]
@clarud Yes, you may widen the scope of the error message as given in the 2nd image.
